Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -150,6 +129,11 @@ if ($fabricInstallerSecret){
     Write-DosMessage -Level "Information" -Message "Fabric.Installer clientSecret has been created."
 }
 
+# Call the Idpss powershell script
+.\Install-IdentityProviderSearchService.ps1 -certificates $($selectedCerts)
```

Comment:
If we already know the credential, should we pass it to the IdPSS script? I think its OK for IdPSS and Identity to run as the same user.